02-spatial-data.Rmd

```diff
@@ -913,6 +912,11 @@ The solution is to set the correct units with the **units** package:
 units::set_units(st_area(nigeria), km^2)
 ```
 
+<!-- Is that right? I mean, the units DESCRIPTION says "Support for measurement units in R vectors, matrices and arrays". Since raster datasets are just matrixes, units might be easily used with them?-->
```

Comment:
@Robinlovelace What do you think?

Reply:
Yes the words below are correct: raster pixel sizes are not given attributes defined by **units**. However, I would replace your line [917](https://github.com/Robinlovelace/geocompr/pull/79/files/dc9edbdb7be85fdb1aa3985da184faf35bbc62bd#diff-237a305a98debc5d406dc19d416f4ce7R917) with the following because **raster** should support **units** not vice versa as currently implied:

```
However, so far **sf** is the only spatial package that supports units, meaning that people working on raster data should approach changes in the units of analysis (e.g. converting pixel widths from imperial to decimal units) with care.
```

Reply:
That also incorporates line 918. I think it would be good to show an example where we calculate the width / height / area of a pixel from a raster dataset already used in this chapter.

Reply:
I adapted your wording, and will think of a raster use case tomorrow.

Reply:
ok, I added a raster unit example